source/common/filesystem/inotify/watcher_impl.cc

```diff
@@ -11,6 +9,7 @@
 
 #include "common/common/assert.h"
 #include "common/common/utility.h"
+#include "common/filesystem/watcher_impl.h"
```

Comment:
I'm guessing the new inotify prefix breaks the header format script (since this passed tests). I think it's probably not worth worrying about but we can potentially make a note to try to make the script deal with this somehow (by having it understand the strip stuff).